cypress.config.ts

```diff
@@ -0,0 +1,16 @@
+import { defineConfig } from 'cypress'
+
+export default defineConfig({
+  chromeWebSecurity: false,
+  defaultCommandTimeout: 15000,
+  retries: {
+    runMode: 2,
+    openMode: 0,
+  },
+  video: false,
+  e2e: {
+    setupNodeEvents(on, config) {
+      return require('./cypress/plugins/index.ts').default(on, config)
```

Comment:
This is only a question to help my understanding and not a change request, but I was wondering if there was a reason to why we import the file here rather than move the code over like below. Do we use the plugins/index file somewhere else as well?
```
setupNodeEvents(on, config) {
     on('file:preprocessor', webpackPreprocessor({
         webpackOptions: webpackConfig
     }));
 }
```

Reply:
Great spot and question!

You're right - there is no technical reason. The v10 migration script added the requires.

When I did the migration I did attempt to change over to imports but ran into some tricky TypeScript errors so left this as a follow up. I should have added a TODO. When we next touch this let's pick this up.